src/test/groovy/unit/org/pih/warehouse/core/LocalizationUtilTest.groovy

```diff
@@ -6,35 +6,52 @@
 * By using this software in any fashion, you are agreeing to be bound by
 * the terms of this license.
 * You must not remove this notice, or any other, from this software.
-**/ 
+**/
 package org.pih.warehouse.core
 
+import org.junit.Test
 import org.pih.warehouse.LocalizationUtil;
 
 
 class LocalizationUtilTest {
 
 	void testGetLocalizedString() {
-	
+
 		Locale locale = new Locale("en","US")
-	
+
 		assert LocalizationUtil.getLocalizedString("Default Value",locale) == "Default Value"
-		
+
 		assert LocalizationUtil.getLocalizedString("Default Value|fr:French Value|es:Spanish Value",locale) == "Default Value"
-		
+
 		locale = new Locale("fr")
-		
+
 		assert LocalizationUtil.getLocalizedString("Default Value|fr:French Value|es:Spanish Value", locale) == "French Value"
-		
+
 		locale = new Locale("es")
-		
+
 		assert LocalizationUtil.getLocalizedString("Default Value|fr:French Value|es:Spanish Value", locale) == "Spanish Value"
-					
-	}	
 
-	void testGetDefaultString() {		
+	}
+
+	void testGetDefaultString() {
 		assert LocalizationUtil.getDefaultString("Default Value|fr:French Value|es:Spanish Value") == "Default Value"
 	}
+
+	@Test
+	void testIsSpanishMexico() {
+		Locale spanishMexico = new Locale("es", "MX")
+
+		Locale english = new Locale("en")
+
+		Locale spanish = new Locale("es")
+
+
+		assert LocalizationUtil.isSpanishMexico(spanishMexico)
+
+		assert !LocalizationUtil.isSpanishMexico(english)
+
+        assert !LocalizationUtil.isSpanishMexico(spanish)
+	}
```

Comment:
FYI: it just looks weird on github. Locally the formatting looks fine:
![Screenshot from 2024-07-02 10-16-25](https://github.com/openboxes/openboxes/assets/93163821/0c7ae5be-6a4e-4762-ac24-b1931c8bff42)